mon-pix/app/components/challenge-item-generic.js

```diff
@@ -101,6 +101,10 @@ const ChallengeItemGeneric = Component.extend({
       }
     },
 
+    resumeAssessment() {
+      return this.resumeAssessment(this.assessment);
```

Comment:
C'est quoi l'intention de cet appel récursif ?

Reply:
Ce n'est pas un appel récursif ; d'ailleurs, la signature n'est pas la même. Le `this.resumeAssessment(this.assessment)` vient de la route. Cf. `validateAnswer` et `this.answerValidated`. 
C'est vrai que ça peut porter à confusion, je vais renommer une des deux méthodes.

Reply:
L'IDE le considérait comme un appel récursif, je me suis demandé si c'était pas une souplesse de javascript / Ember 🤷‍♂ 
Il doit faire des associations par nom. Du coup, un renommage me semble être une bonne approche pour clarifier ça 👍

Reply:
Dans `challenge.js`, je pense que tu peux renommer en `resume(assessment)` plutôt que `resumeAssessment(assessement)` qui amène une redondance. 

D'ailleurs, le fait que l'on parle de `resumeAssessment` dans un challenge me dérange un peu. Dans ce cas, on resume un challenge même si techniquement, ça revient à resume un assessment 🤔

Reply:
Ça pourrait être par exemple 
```js
resume() {
	return this.resumeAssessment(this.assessment);
}
```

Ou 
```js
resumeChallenge() {
	return this.resumeAssessment(this.assessment);
}
```
ou
```js
resumeChallenge() {
	return this.resume(this.assessment);
}
```